notes/2020/2020-05-21.md

```diff
@@ -0,0 +1,93 @@
+# 2020-May-21 ESLint TSC Meeting Notes
+
+## Transcript
+
+[`transcripts/5-21-2020.md`](../../transcripts/5-21-2020.md)
```

Comment:
I _think_ this relative link will work, though I'll need to update it depending on https://github.com/eslint/tsc-meetings/pull/178#pullrequestreview-416623075.

Reply:
```suggestion
[`notes/2020/2020-05-21-transcript.md`](2020-05-21-transcript.md)
```